internal/network/command.go

```diff
@@ -136,3 +143,13 @@ func (c *command) getNetworks() ([]networkingv1.NetworkingV1Network, error) {
 
 	return c.V2Client.ListNetworks(environmentId)
 }
+
+func AddConnectionTypesFlag(cmd *cobra.Command) {
```

Comment:
I assume these aren't going to be used outside of the `network` package.
```suggestion
func addConnectionTypesFlag(cmd *cobra.Command) {
```